RELEASE.md

```diff
@@ -2,12 +2,8 @@
 
 Releasing the project requires these steps:
 
-0. Set the version number in the code
-1. BUILD-COMMAND
-2. Use a GitHub [project release][github-release-url] to release the project and tag (be sure it follows [semver][semantic-versioning])
-3. PACKAGE-COMMAND and/or RELEASE-COMMAND
-4. Update `master` to a new minor version
-
-[project-url]: https://github.com/cerner/cucumber-forge-report-generator/
-[semantic-versioning]: http://semver.org/
-[github-release-url]: https://help.github.com/articles/creating-releases/
+0. Upgrade dependency versions as necessary. (See: `npm audit`)
+1. Run `npm version *major|minor|patch*` (this project uses [semantic versioning](http://semver.org/))
+2. Push new commit to master and push the newly created tag (this will trigger a Travis build)
+3. Wait for the Travis build to complete
+4. [Create a new release on GitHub](https://help.github.com/articles/creating-releases/) for the tag and add changelog information for the changes included in the release
```

Comment:
```suggestion
0. Upgrade dependency versions as necessary. (See: `npm audit`)
0. Run `npm version *major|minor|patch*` (this project uses [semantic versioning](http://semver.org/))
0. Push new commit to master and push the newly created tag (this will trigger a Travis build)
0. Wait for the Travis build to complete
0. [Create a new release on GitHub](https://help.github.com/articles/creating-releases/) for the tag and add changelog information for the changes included in the release
```